cartridge/lua-api/failover.lua

```diff
@@ -168,14 +173,29 @@ local function promote(replicaset_leaders)
     local _, err = rpc.call(
             'failover-coordinator',
             'appoint_leaders',
-            {replicaset_leaders},
+            {replicaset_leaders, opts},
```

Comment:
Seems it will be ignored (https://github.com/tarantool/cartridge/pull/985/files#diff-1bf8cd740a6ca039e5fc4756af8797b7R326)